src/frontend/js/widgets/CourseProductItem/components/PurchaseButton/index.tsx

```diff
@@ -70,15 +81,20 @@ const PurchaseButton = ({ product, disabled }: PurchaseButtonProps) => {
             aria-description={intl.formatMessage(messages.callToActionDescription, {
               product: product.title,
             })}
-            disabled={!hasAtLeastOneCourseRun}
+            disabled={!isPurchasable}
           >
             {product.call_to_action}
-          </button>
+          </Button>
           {!hasAtLeastOneCourseRun && (
             <p className="product-item__no-course-run">
               <FormattedMessage {...messages.noCourseRunToPurchase} />
             </p>
           )}
+          {!hasAtLeastOneRemainingOrder && (
```

Comment:
We may replace this by : 
```js
{hasAtLeastOneCourseRun && !hasAtLeastOneRemainingOrder && (
```
it depend if we want both messages and if not, what message do we want to prioritize.

Reply:
I think we want first to display feedback message to inform there is one target course without course run (`hasAtLeastOneCourseRun`) and second display feedback message to inform there isn't available seat (`!hasAtLeastOneRemainingOrder`)

Reply:
I would display only one message at a time with the priority I described above.